api/src/settings.py

```diff
@@ -37,9 +37,12 @@ class Settings(BaseSettings):
     UAZ_THRESHOLD: str = ""
     UAZ_HITS: str = ""
 
+    DEBUG: bool = False
+
     PLUGINS: Dict[str, str] = {
         # "my_plugin": "plugin_module.MyPlugin",  # Where plugin_module.MyPlugin is an importable dotted path and MyPlugin
-                                                  # is a subclass of utils.PluginInterface
+        # is a subclass of utils.PluginInterface
+        # "causemos": "src.plugins.causemos.CausemosPlugin",
```

Comment:
I assume this is important to be off for dev, and overwritten on prod by envfile config 👍🏼